datasets/sk/public_officials/sk_public_officials.yml

```diff
@@ -0,0 +1,50 @@
+title: Slovakia Public Officials
+prefix: sk-po
+entry_point: crawler.py
+coverage:
+  frequency: weekly
+  start: 2024-12-23
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+summary: >
+  List of public officials in Slovakia, including their names, positions,
+  and other relevant information.
+description: |
+  The dataset contains information about public officials in Slovakia, including
+  their names, positions, and other relevant information. The data is collected
+  from the official website of the National Council of the Slovak Republic.
+publisher:
+  name: National Council of the Slovak Republic
+  acronym: NR SR
+  description: |
+    National Council of the Slovak Republic is the sole constitutional and legislative body 
+    of the Slovak Republic. It is the body of state authority and the status of other state 
+    bodies in Slovakia is derived from its primary status. As an elected body it represents 
+    the sovereignty of the state and of the citizens. It has an important role in the formation 
+    of the Slovak Republic as a modern and democratic state, as well as in the implementation of 
+    social and ecologically oriented market economy.
+    [Source: Official website](https://www.nrsr.sk/web/?sid=nrsr/poslanie)
+  url: https://www.nrsr.sk/web/
+  official: true
+  country: sk
+url: https://www.nrsr.sk/web/Default.aspx?sid=vnf%2fzoznam&ViewType=2
+data:
+  url: https://www.nrsr.sk/web/Default.aspx?sid=vnf%2fzoznam&ViewType=2
+  format: HTML
+  lang: slk
+
+assertions: 
+  min: 
+    schema_entities: 
+      Person: 2000
+  max: 
+    schema_entities: 
+      Person: 3000
+lookups:
+  type.name:
+    options:
+      - contains: "verejný funkcionár, ktorý nie je uvedený v písmenách a) až zo)"
+        value: null
+  type.string: 
+    options: 
+      - contains: "verejný funkcionár, ktorý nie je uvedený v písmenách a) až zo)"
+        value: null
```

Comment:
this will make the Position:name prop empty for the positions that contain this. This won't just strip out the matching bit. So I think we'll have to do this in Python